source/unified-test-format/schema-1.6.json

```diff
@@ -0,0 +1,466 @@
+{
```

Comment:
I had to add a new schema version 1.6 for `hasServerConnectionId` per the rules on UTF schema version bumps [here](https://github.com/mongodb/specifications/blob/master/source/unified-test-format/unified-test-format.rst#impact-of-spec-changes-on-schema-version).